### DISCUSSION TITLE
Upcoming 4.7.0 release

### DISCUSSION THREAD

**Discussion by eakraly** [coturn/coturn#1612] [Announcements]
The plan for upcoming 4.7.0 release includes a few breaking changes:
- Remove support for old openssl versions (<1.1.1)
- Make SOFTWARE_ATTRIBUTE optional by default (reverse current behavior to reduce amplification factor and remove the need to pass additional configuration for production usage)
- Remove configuration options that have been deprecated for some time
- Drop builds on old OS versions (Ubuntu < 20.04, older macOS, amazonlinux2)
- (additional breaking changes TBD)

**Comment by jonesmz** [comment]
There all look great!
